templates/fields.html.twig

```diff
@@ -75,9 +75,20 @@
                 'type': 'url',
                 'add_field_html': ext_link
             })) }}
+
     {% elseif type == 'textarea' %}
         {{ macros.textareaField(name, value, label, field_options) }}
 
+    {% elseif type == 'richtext' %}
+        {{ macros.textareaField(name, value, label, field_options|merge({
```

Comment:
By default `enable_images` will be `true`. I did not tested it yet, but I am pretty sure that pasted images will remain in base64 and will not be converted into documents. I think a call to `CommonDBTM::addFiles()` should be added somewhere to correctly handle uploaded images.